timeseries/src/autogluon/timeseries/models/sktime/abstract_sktime.py

```diff
@@ -67,21 +67,32 @@ def __init__(
             hyperparameters=hyperparameters,
             **kwargs,
         )
-        self.skt_forecaster: Optional[BaseForecaster] = None
+        self.sktime_forecaster: Optional[BaseForecaster] = None
         self._fit_index: Optional[pd.Index] = None
 
-    def _get_skt_forecaster(self, sp: Optional[int] = None) -> BaseForecaster:
-        """Return the sktime forecaster object for the model"""
-        params = self._get_model_params().copy()
-        if sp is not None and sp > 1:
-            params["sp"] = sp
-            params["seasonal"] = "add"
+    def _get_sktime_forecaster_init_args(
+        self, model_params: Dict[str, Any], min_length: int, inferred_period: int = 1
```

Comment:
Is `model_params` an okay name here? The logic here is that we received them by calling `self._get_get_model_params().copy()` @canerturkmen

Reply:
cool